pkg/assembler/backends/inmem/certifyLegal_test.go

```diff
@@ -485,7 +485,7 @@ func TestLegal(t *testing.T) {
 	ctx := context.Background()
 	for _, test := range tests {
 		t.Run(test.Name, func(t *testing.T) {
-			b, err := inmem.GetBackend(nil)
+			b, err := backends.Get("inmem", nil, nil)
```

Comment:
Looks good! Just need to update the arango tests with these also

Reply:
done